packages/cysync-core/src/constants/routes/onboarding.ts

```diff
@@ -31,12 +39,23 @@ export const onboarding = {
     name: 'onboarding-card-authentication',
     path: '/onboarding/card-auth',
   },
-
   congratulations: {
     name: 'onboarding-congratulations',
     path: '/onboarding/congo',
   },
 } as const;
 
+export const utilityRoutes = {
+  back: {
+    name: 'onboarding-back',
+    path: '?back=true',
+  },
+  forward: {
+    name: 'onboarding-front',
+    path: '?forward=true',
+  },
+} as const;
```

Comment:
These are not `routes`, they are query parameters, hence they should not be defined under `routes`.